crates/c_api/src/extern.rs

```diff
@@ -89,6 +99,10 @@ pub extern "C" fn wasm_extern_as_table(e: &mut wasm_extern_t) -> Option<&mut was
 ///
 /// Returns `None` if `e` is not a [`wasm_table_t`].
 #[no_mangle]
+#[cfg_attr(
+    feature = "prefix-symbols",
+    export_name = "wasmi_wasm_extern_as_table_const"
+)]
```

Comment:
This should use the `#[cfg_attr(feature = "prefix-symbols", wasmi_c_api_macros::prefix_symbol)]` attribute. Probably an oversight due to text replacement without multi-lines.